app/components/partner/client-info/template.hbs

```diff
@@ -26,43 +26,23 @@
         </div>
       </div>
       {{#if this.partner.access.view_plans}}
-        <div local-class="row">
-          <div local-class="key">{{t 'paymentPlan'}}</div>
-          <div local-class="value">
-            <div local-class="plan" {{did-insert (perform this.getClientPlan)}}>
-              {{#if this.getClientPlan.isRunning}}
-                <div local-class="plan-loading">{{fa-icon "circle-o-notch fa-spin"}}</div>
-              {{else}}
-                <div local-class="plan-type {{if this.clientPlan.limitedScans 'per-scan' 'per-app'}}">{{t (if  this.clientPlan.limitedScans 'perScan' 'perApp')}}</div>
-                <div local-class="plan-status">
-                  {{#if this.clientPlan.limitedScans}}
-                    <strong class="margin-r-h">
-                      {{this.clientPlan.scansLeft}}
-                      {{t 'pluralScans' itemCount=this.clientPlan.scansLeft}}
-                    </strong>
-                    {{t 'remaining'}}
-                  {{else}}
-                    <strong class="margin-r-h">
-                      {{this.clientPlan.projectsLimit}}
-                      {{t 'pluralApps' itemCount=this.clientPlan.projectsLimit}}
-                    </strong>
-                    <span local-class="expiry-date {{if this.clientPlan.isPaymentExpired 'expiry-date-expired'}}">
-                      {{t (if this.clientPlan.isPaymentExpired 'expiredOn' 'expiresOn')}} {{day-js date=this.clientPlan.expiryDate format="DD MMM YYYY"}}
-                    </span>
-                  {{/if}}
-                </div>
-              {{/if}}
-            </div>
-          </div>
+        <div {{did-insert (perform this.getClientPlan)}}>
+          {{#if this.getClientPlan.isRunning}}
+            <div local-class="plan-loading">{{fa-icon "circle-o-notch fa-spin"}}</div>
+          {{else}}
+            <Partner::ClientInfoPlan @clientPlan={{this.clientPlan}} data-test-payment-plan />
+          {{/if}}
         </div>
       {{/if}}
       <div local-class="row">
         <div local-class="key">{{t 'pluralorganizationOwner' itemCount=@client.ownerEmails.length}}</div>
         <div local-class="value">
           <div local-class="client-owner">
```

Comment:
A new component created under modals, since the email list is rendered by a modal. Also, the email list will be more flexible like the number of default emails shown (comma separated) controlled by injector but set 1 as default, etc.,

Reply:
Wouln't it be better if `emails-list` was generic, then it will be able to process any list. Eg:
```js
<PlusNList
    enableLinks=<bool defaults to false>
    list=<array of objects [{'label': <str>, 'link': <str>}, ...]>
    suffix=<optional string, suffix to count (eg: +n more, +n members, +n tags, etc.)>
    showLineNumbers=<optional bool, defaults to false>
    modalTitle=<str>
/>
```
instead of:
```js
<Modals::EmailList
    @emails={{@client.ownerEmails}}
    @headerTitle='Client owner emails'
/>
```
`plus-n-list` component will accept a list of text labels, and optionally links associated with each label. This can be used for any list. Example usage:

```js

<PlusNList
    enableLinks=false
    list={{this.ownerEmailList}}
    suffix="owners"
    showLineNumbers=true
    modalTitle='Client owner emails'
/>
```
where `ownerEmailList` is of the format `[{'label': 'owner@example.com', link: ''}, ...]`
Point of links is that we can make the tags optionally clickable in the list

Reply:
I really like the idea of having a more generic component where any list could be shown with + items. Regarding the introduction of links, there could be additional things has to be considered/figured out whether it will be an external link or internal route (which points page inside app) or whether it has to be opened on a separate tab while clicking on it, etc., 
So, for now I think it's better to go ahead without link feature, and could be added in future when required and by defining all the above items! @Praseetha-KR

Reply:
Ya makes sense 👍 well, in that case, may be we can make a dedicated component for that something like `PlusNLinkList`

Reply:
Yeah, it could be a separate PR 👍

Reply:
included in this PR itself